### ISSUE TITLE
gretty not deploying the war for integrationTestTask

### ISSUE THREAD

**Issue by techpy** [akhikhl/gretty#271]
Hi,

This is not an issue per se and more a lack of knowledge on my end. I posted the issue initially on StackOverflow - unfortunately did not get any response. Hoping somebody here can help me.

The issue is described on StackOverflow:

http://stackoverflow.com/questions/37841760/gretty-gradle-plugin-version-1-2-4-how-to-ensure-war-is-deployed-when-using-in

Any idea how do I ensure my war is deployed by Gretty. I have checked the build/serverBaseDir_jetty9/webapps-exploded/<myapp> (my servlet container is jetty9) and I can see that the war is not deployed.

My gradle file looks as follows:

apply plugin: 'java'
apply plugin: 'eclipse'
apply plugin: 'war'
apply plugin: 'org.akhikhl.gretty'

....
war {
	
	webXml = file("src/main/web/WEB-INF/web.xml")
	
    archiveName = "$project.name"+".war"
 
    classpath = classpath - sourceSets.main.output
 
    from (jar) {
        into 'WEB-INF/lib'
    }
 
    includeEmptyDirs = false
 
    entryCompression = ZipEntryCompression.DEFLATED
}

gretty {
    port = 8080
    contextPath = 'hello'
    servletContainer = 'jetty9'
    loggingLevel = "DEBUG"
    logDir = "$buildDir/logs"
    
    integrationTestTask = 'test'
}

**Comment by andersaa** [comment]
I don't think this is supported in Gretty at the moment. Gretty starts the servlet container against the compiled classes, not against the war file. It does not execute the war task. Have a look at the "Uses WAR" column here:

http://akhikhl.github.io/gretty-doc/Gretty-tasks.html

If you can make your application run by executing ./gradlew appStart then the integrationTest task should work as expected too. You might need to put web.xml in src/main/webapp/WEB-INF or figure out how to configure its location outside the war {} configuration.

It would be very nice to have this option in Gretty though!
